test/e2e/helloworldsampleapp_test.go

```diff
@@ -0,0 +1,76 @@
+// +build e2e
```

Comment:
I would drop "helloworldsample" from the name, and use something like "sampleapps_test.go".

Reply:
done